mkosi/resources/mkosi-tools/mkosi.conf.d/debian-kali-ubuntu/mkosi.conf

```diff
@@ -17,6 +17,7 @@ Packages=
         libseccomp2
         libtss2-dev
         policycoreutils
+        python3
```

Comment:
We should probably also do `python-is-python3` here, since then we have the `python -> python3` symlink across all distros, I think.